recipes/open62541/all/test_package/test_package.c

```diff
@@ -12,64 +12,10 @@
 #include "foo_flt_nodeids.h"
 #include "namespace_foo_flt_generated.h"
 #include <open62541/plugin/network.h>
-#include <signal.h>
```

Comment:
We usually don't like test packages that do _too much_, specially one they try to use network connectivity, open audio devices etc.

This simplifies it to is maximum, but happy to walk back some of the changes.

For example, I've noticed that the test package conanfile.py requires on another package entirely different. Is this because the library is _always_ used with it? Or was the test package using it just because the code here was using it? If so removing the requies and looking for a simpler test would work much better :)

Reply:
1. The test package has been rewritten by another contributor to test the functionality more thoroughly I wanted to discuss the changes with him before changing the test case, but sadly he never responded. I also discussed this topic in PR 
#14537 in [this comment](https://github.com/conan-io/conan-center-index/pull/14537#discussion_r1044385010) suggesting a compromise, but it was never resolved, before the PR was merged in. To ensure that base functionality works and options are passed, I would like to at least create the default configuration and add it to the server. Loading the server interface symbols is not good enough from my experience to ensure that the package is working properly.
2. The test package uses auto-generated nodes, described by another package to test that this functionality works. It gave me some problems in the past, so I would like to keep testing that functionality to ensure that it works with each release.

Reply:
hi @RubenRBS, did you get a chance to review my comment for the `test_package.c` changes?

Reply:
Yes! Sorry. Approving now, thanks for the extra insgiht, really useful :)

Reply:
Feel free to ignore this change :)